JavaScript/Topics/7.Promises/README.md

```diff
@@ -0,0 +1,6 @@
+# Promises In Javascript
+
+### Video Lecture On Promises
+Video-[promises](https://youtu.be/ASfaBnNbeDc)
```

Comment:
It should be like this.
`Video Link: [Promises in JavaScript](https://youtu.be/ASfaBnNbeDc)`
Video Link: [Promises in JavaScript](https://youtu.be/ASfaBnNbeDc)

Also change the video title to `Promises in JavaScript`